DXMainClient/DXGUI/GameClass.cs

```diff
@@ -69,6 +69,15 @@ protected override void Initialize()
             Window.Title = string.IsNullOrEmpty(windowTitle) ?
                 string.Format("{0} Client", MainClientConstants.GAME_NAME_SHORT) : windowTitle;
 
+            {
+                string developBuildTitle = "(Development Build)".L10N("Client:Main:DevelopmentBuildTitle");
+
+#if DEVELOPMENT_BUILD
+                if (ClientConfiguration.Instance.ShowDevelopmentBuildWarnings)
+                    Window.Title += " " + developBuildTitle;
+#endif
```

Comment:
Is it needed to translate parentheses? Maybe
```suggestion
                string developBuildTitle = "Development Build".L10N("Client:Main:DevelopmentBuildTitle");

#if DEVELOPMENT_BUILD
                if (ClientConfiguration.Instance.ShowDevelopmentBuildWarnings)
                    Window.Title += $" ({developBuildTitle})";
#endif
```

would work better?